src/Command/addBlock.php

```diff
@@ -28,14 +28,28 @@ public function handle()
         $installed_modules = app('module.collection')->installed();
         $installed_modules = $installed_modules->merge(app('extension.collection')->installed());
 
-        $views = "";
+        $views = array();
         $params = $this->params;
+        $requiresOrdering = array();
         foreach ($installed_modules as $item) {
             if (file_exists($item->path . "/resources/views/" . $this->location . ".twig")) {
-                $views .= view($item->namespace . '::' . $this->location, compact('params'));
+                if (isset($this->params['_ORDER_']) &&
+                    array_search($item->namespace, $this->params['_ORDER_'], true) !== false) {
```

Comment:
why do we make array search here, and why duplicate of it ?

Reply:
The array search is looking for the prefered order that the blocks should come in, so if you added any block to the _ORDER_ array it will get displayed on the top of the other blocks, I am using array_search twice because the first time I am checking if the block exists in the _ORDER_ array, and the second time is to check in which order did it come in the _ORDER_ array just in case you want to put multiple blocks on top and order them as well, I can use array_search once on top but then I will have to use 2 if statments one to check the isset() and the other one is to check the order